benchmark/test_benchmark_jubjub.py

```diff
@@ -0,0 +1,151 @@
+import asyncio
+from pytest import mark
+from honeybadgermpc.progs.jubjub import SharedPoint, share_mul
+from honeybadgermpc.progs.mixins.share_arithmetic import (
+    BeaverMultiply, BeaverMultiplyArrays, InvertShare, InvertShareArray, DivideShares,
+    DivideShareArrays, Equality)
+from honeybadgermpc.elliptic_curve import Jubjub, Point
+from honeybadgermpc.mpc import TaskProgramRunner
+
+
+CONFIG = {
+    BeaverMultiply.name: BeaverMultiply(),
+    BeaverMultiplyArrays.name: BeaverMultiplyArrays(),
+    InvertShare.name: InvertShare(),
+    InvertShareArray.name: InvertShareArray(),
+    DivideShares.name: DivideShares(),
+    DivideShareArrays.name: DivideShareArrays(),
+    Equality.name: Equality()
+}
+
+TEST_PREPROCESSING = [
+    'rands', 'triples', 'bits'
+]
+
+TEST_CURVE = Jubjub()
+TEST_POINT = Point(5, 6846412461894745224441235558443359243034138132682534265960483512729196124138, TEST_CURVE)  # noqa: E501
+
+n, t = 4, 1
+k = 10000
+COUNT_MAX = 2
+
+
+def test_benchmark_shared_point_add(test_preprocessing, benchmark):
+    async def _prog(context):
+        result = SharedPoint.from_point(context, TEST_POINT)
+        result = result.add(result)
+        await result.open()
+
+    for kind in TEST_PREPROCESSING:
```

Comment:
Is it possible that we can move the code from below this line to the end of this method in a separate method and then reuse it?

Reply:
Good idea-- I'll add a fixture for it.